docs/reference/sql/endpoints/jdbc.asciidoc

```diff
@@ -43,9 +43,9 @@ from `artifacts.elastic.co/maven` by adding it to the repositories list:
 [float]
 === Setup
 
-The driver main class is `org.elasticsearch.xpack.sql.jdbc.jdbc.JdbcDriver`. 
+The driver main class is `org.elasticsearch.xpack.sql.jdbc.EsDriver`. 
```

Comment:
I think it is a shame to change the class name, but I think if we're going to do it then now is the time.